### DISCUSSION TITLE
blank screen on android 9 webview is 74.0.3729.136.

### DISCUSSION THREAD

**Discussion by swordfly1979** [quasarframework/quasar#14597] [CLI - Capacitor / Cordova modes] [answered]
My mobile phone is android9. The version of android system webview is 74.0.3729.136. The running screen is blank. An error is reported as follows: Uncaught (in promise) TypeError: Promise allSettled is not a function at client-entry.js:141:20

**Comment by swordfly1979** [comment, marked as answer]
error
![image](https://user-images.githubusercontent.com/44988237/194991258-ca32f282-7346-4dd2-8497-99f2af33c4ac.png)

**Comment by rstoenescu** [comment]
@swordfly1979 
Backward compatibility with older webviews will be available out of the box in q/app-vite 1.1.3 and q/app-webpack 3.6.2
Just pushed the changes.
